docs/kqueen.rst

```diff
@@ -428,6 +428,32 @@ To manually add an existing Kubernetes cluster:
 As a result, the Kubernetes cluster will be attached in a read-only mode.
 
 
+Kqueen UI Cluster actions:
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Currently supported cluster operations:
+
+.. code-block:: text
+
+   Before cluster provision:
+
+1. Set the ``Name of cluster``.
+2. Set the ``Node count``.
+3. Set the deployment ``Zone``.
+4. Set the ``Machine type``.
+5. Set the ``Network range`` for cluster (Currently GKE supported only).
+6. Set the ``Network policy`` for managing cluster network policies (Currently GKE supported only).
+
+.. code-block:: text
+
+   After cluster provision:
```

Comment:
and here you are describing operations with existent clusters

Reply:
agree, but prefer to wait mzlatkova review too,